osl/namespace_linux.go

```diff
@@ -498,3 +514,79 @@ func (n *networkNamespace) Restore(ifsopt map[string][]IfaceOption, routes []*ty
 
 	return nil
 }
+
+// Checks whether IPv6 needs to be enabled/disabled on the loopback interface
+func (n *networkNamespace) checkLoV6() {
+	var (
+		loV6   = false
+		action = "disable"
+	)
+
+	n.Lock()
+	for _, iface := range n.iFaces {
+		if iface.AddressIPv6() != nil {
+			loV6 = true
+			action = "enable"
+			break
+		}
+	}
+	n.Unlock()
+
+	if err := setIPv6(n.path, "lo", loV6); err != nil {
```

Comment:
I'd need to keep a variable in the networkNamespace structure for that. Let me make the change. Thanks